hadoop-hdds/docs/content/design/volume-management.md

```diff
@@ -125,11 +125,12 @@ Note: Sanjay is added to the authors as the original proposal of this approach.
  * `bucket link` operation creates a link bucket.  Links are like regular buckets, stored in DB the same way, but with two new, optional pieces of information: source volume and bucket.  (The bucket being referenced by the link is called "source", not "target", to follow symlink terminology.)
  * Link buckets share the namespace with regular buckets.  If a bucket or link with the same name already exists, a `BUCKET_ALREADY_EXISTS` result is returned.
  * Link buckets are not inherently specific to a user, access is restricted only by ACL.
- * Link buckets themselves do not store ACLs. All ACLs operations (eg. addacl/getacl/setacl/removeacl) and checks on the link bucket will be applied to the source bucket.
+ * Link buckets retain their owner ACLs, which are inherited from the default ACLs of their volume.
+ * All add/set/remove ACL operation proxy to the source bucket. Getacl operation of a link bucket shows source bucket by default (Use option to display ACLs of the link bucket itself).
  * Links are persistent, ie. they can be used until they are deleted.
  * Existing bucket operations (info, delete, ACL) work on the link object in the same way as they do on regular buckets.  No new link-specific RPC is required.
  * Links are followed for key operations (list, get, put, etc.).  Read permission on the link is required for this.
```

Comment:
Links are followed for key operations (list, get, put, etc.). Read permission on the link is required for this. -> 

Links are followed for key operations (list, get, put, etc.). Read permission on the source bucket is required for this.